bindings/experimental/distrdf/test/test_headnode.py

```diff
@@ -209,6 +225,11 @@ def test_num_entries_with_ttree_arg(self):
             v[i] = 1  # Change the vector element to 1
             tree.Fill()  # Fill the tree with that element
 
+        f.Write()
+
         hn = create_dummy_headnode(tree)
 
-        self.assertEqual(hn.get_num_entries(), 4)
+        self.assertEqual(hn.tree.GetEntries(), 4)
```

Comment:
what are you testing here?

Reply:
This and the above tests in class `NumEntriesTest` were previously testing that the method `get_num_entries` of `TreeHeadNode` was returning the correct amount. Now, they test that the attribute `tree` of the class is the correct TTree/TChain that we expect by retrieving the number of entries.

Reply:
I may agree that it's not a super useful test, but I didn't want to remove it as a first step